packages/eslint-plugin-baseui/__tests__/no-deep-imports.js

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright (c) 2018-2020 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+const RuleTester = require('eslint').RuleTester;
```

Comment:
nit: I think we mostly use destructuring across the code base - may worth updating here too